### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by janani-sridhar**
﻿# Pull Request Template
### Overview
Updated the README.md file with additional information on Augur.

### Changes proposed for this pull request
N/A

### Notes
N/A

### Testing Instructions
N/A

### Examples (if necessary)
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,15 +197,7 @@ base 10\*\*18).
...
-## General information about Serpent
-
-- [The Serpent wiki](https://github.com/ethereum/wiki/wiki/Serpent)
-- [The Serpent guide](https://www.cs.umd.edu/~elaine/smartcontract/guide.pdf) (PDF)
-
-## General information about Augur
-
-- [A Roadmap For Augur and What’s Next](https://medium.com/@AugurProject/a-roadmap-for-augur-and-whats-next-930fe6c7f75a)
-- [Augur Master Plan](https://medium.com/@AugurProject/augur-master-plan-42dda65a3e3d)
+Also note, Mist uses a different contract abi style than usual, to convert use the following regex \((.\*?)\) and replace with nothing [except don't do it for events].
```
Comment (beaugunderson) [inline README.md]: no need to add this bit back as we decided previously to leave it out :)

Comment (janani-sridhar) [inline README.md]: You mean the part about Mist, or the part about the roadmap for augur?

Comment (joeykrug) [inline README.md]: I'd leave it back in tbh, I wasted hours once figuring this out, it's still a non-fixed bug in mist, etc. Although mist still doesn't even support contract bytecode uploads (it got broke about a year ago) so perhaps it's not too important

Comment (beaugunderson) [inline README.md]: yes, meant the mist-specific part; and cool re: leaving it there 👍

**Pull request merged**